docs/topics/rules-testing-junit.adoc

```diff
@@ -15,7 +15,7 @@ Once a test rule has been created, it can be analyzed as part of a JUnit test to
 
 .Creating the JUnit test configuration
 
-The following instructions detail creating a JUnit test using the Red Hat CodeReady Studio. When using a different IDE it is recommended to consult your IDE's documentation for instructions on creating a JUnit test.
+The following instructions detail creating a JUnit test using Eclipsex. When using a different IDE it is recommended to consult your IDE's documentation for instructions on creating a JUnit test.
```

Comment:
"Eclipsex" or "Eclipse" ? I Googled Ecipsex and found nothing that looks appropriate, so please double-check.

Reply:
Fixed